services/121-service/src/fsp-integrations/integrations/intersolve-visa/interfaces/partials/contact-information.interface.ts

```diff
@@ -1,4 +1,5 @@
 export interface ContactInformation {
+  readonly name: string;
   readonly addressStreet: string;
   readonly addressHouseNumber: string;
```

Comment:
I am happy that we went this route, adding the name here instead of 2 seperate contact informations